### ISSUE TITLE
Add an option for enabling Virtual Space

### ISSUE THREAD

**Issue by m6502** [notepad-plus-plus/notepad-plus-plus#11443]
### Description of the Issue
Scintilla supports Virtual Space, but there's no option for enabling it.

### Steps to Reproduce the Issue
Launch Notepad++. Be confused while working with it if you are used to Virtual Space. Unsuccessfully search for an option to free the caret from the tyranny of present characters. End up using some hacky solution.

### Expected Behavior
Clicking a nice "Enable Virtual Space" checkbox in General -> Editing.

### Actual Behavior
There's no "Enable Virtual Space" checkbox in General -> Editing.

##

Opening this issue as the prerequisite for an incoming pull request :-)

**Comment by m6502** [comment]
So, I have already implemented this in my local copy of the code, and looks like it's working OK. But as this is the first time I contribute to Notepad++ I'm a bit afraid of breaking it, so I would appreciate some feedback on two things. If any of these has already been answered before I would honestly appreciate a link to the discussion. 

First of all, I have had to add a checkbox and I'm not sure what is the "official" method of doing this. I first tried to simply use Visual Studio 2022 and it complained about not being the original editor used to create the file and warned about the possibility of losing some data. To be honest, I didn't see anything missing but the rc file ended up a little dirty after being saved this way. So I tried again, this time by hand editing the resource file. Is there a better way to do it?

Next question is related about how to handle the text translations for the checkbox. Before creating the pull request I want to ensure the program doesn't break when selecting another language. I see the XML files containing the translations are composed by lines like <Item id="number" name="some text"/> . I'm not really sure where the Item ID is coming from.

I would appreciate a hand on understanding these two issues, thank you :-)
